### PR TITLE
Creating projects section and about me section

### PR DESCRIPTION
**Pull request by a-laarabi**
# What?
Creating projects and about me sections which are necessary in every portfolio in the world

# Why?
Studying

# How?
HTML and CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h4>LET’S CONNECT</h4>
+        <ul class="connect">
+            <li><a href="#"><img src="img/twitter.png" alt="twitter"></a></li>
+            <li><a href="#"><img src="img/Linkedin_icon.png" alt="Linkedin"></a></li>
+            <li><a href="#"><img src="img/medium.png" alt="medium"></a></li>
+            <li><a href="#"><img src="img/github.png" alt="github"></a></li>
+            <li><a href="#"><img src="img/angellist.png" alt="angellist"></a></li>
+        </ul>
+        <section class="see2"><a href="#">Get my resume</a></section>
```
Comment (omarsalem7) [inline index.html:105]: - [ ] Please add button styling: Active, Hover, and Pressed to this button also.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        <h2>About<br>Myself</h2>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <h4>LET’S CONNECT</h4>
+        <ul class="connect">
+            <li><a href="#"><img src="img/twitter.png" alt="twitter"></a></li>
+            <li><a href="#"><img src="img/Linkedin_icon.png" alt="Linkedin"></a></li>
+            <li><a href="#"><img src="img/medium.png" alt="medium"></a></li>
+            <li><a href="#"><img src="img/github.png" alt="github"></a></li>
+            <li><a href="#"><img src="img/angellist.png" alt="angellist"></a></li>
+        </ul>
```
Comment (omarsalem7) [inline index.html:104]: - [ ] Please try to match your design like Figma

| Your design  | Figma design|
| ------------- | ------------- |
| image  | image |

- [ ] Consider fixing the space in right in about me (hint: remove ` min-width: 415px; from work class to fix this issue`)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                        <li class="project_info">Back End Dev</li>
+                        <li class="project_info">2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="project-language">
+                        <li><a href="#">html</a></li>
+                        <li><a href="#">css</a></li>
+                        <li><a href="#">javaScript</a></li>
+                    </ul>
+                    <section class="see"><a href="#">See Project</a></section>
```
Comment (omarsalem7) [inline index.html:31]: - [ ] Kindly follow the button styling: Active, Hover, and Pressed to see the project button, you can find the colors in the Figma template.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @a-laarabi, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near


## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cwaku — APPROVED** [review]
Hi @a-laarabi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**